src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -97,7 +102,6 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
 
   @throws[ConfigurationException]("when no existing readable directory is configured")
   private def getConfiguredDirectory(key: String): File = {
-    // TODO move to Validation?
```

Comment:
`StartupValidation` only contains the `sameMounts` method because it is called from the `Submitter` constructor which is called at startup. [You wanted](https://github.com/DANS-KNAW/easy-deposit-api/pull/128#discussion_r270464204) a separate class for that method.

The question emerged from that change. Somehow I came to the conclusion the answer to the question was no. Various thoughts:

A `sameMount` check implicitly checks existence of directories as it tries to move files between them. `getConfiguredDirectory` does these checks explicitly for clear error messages. The properties object doesn't belong in a validation class.

I tend to let components created by the app-class to do their own validation. The `DepositServlet` doesn't know about other directories than the multipart.location, so it can't do the same-mount check at construction time and the app-class has to take that responsibility.